apiservers/engine/CLI-COMPATIBILITY.md

```diff
@@ -0,0 +1,274 @@
+|--cap-add=[]|Add Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cap-drop=[]|Drop Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cgroup-parent=""|Optional parent cgroup for the container|NO, VIC containers do not run in cgroups|
+|--cidfile=""|Write the container ID to the file|YES|
+|--cpuset-cpus=""|CPUs in which to allow execution (0-3, 0,1)|NO|
+|--device=[]|Add a host device to the container|*diff*|
+|--dns=[]|Set custom DNS servers|YES|
+|--dns-search=[]|Set custom DNS search domains|YES|
+|-e, --env=[]|Set environment variables|YES|
+|--entrypoint=""|Overwrite the default ENTRYPOINT of the image|YES|
+|--env-file=[]|Read in a file of environment variables|YES|
+|--expose=[]|Expose a port or a range of ports|YES|
+|-h, --hostname=""|Container host name|YES|
+|-i, --interactive=false|Keep STDIN open even if not attached|*maybe*|
+|--ipc=""|IPC namespace to use|NO, VIC containers run in separate VMs|
+|-l, --label=[]|Set metadata on the container (e.g., --label=com.example.key=value)|YES|
+|--label-file=[]|Read in a line delimited file of labels|YES|
+|--link=[]|Add link to another container|*maybe*|
+|--log-driver=""|Logging driver for container|YES|
+|--lxc-conf=[]|Add custom lxc options|NO, VIC containers do not run in cgroups|
+|-m, --memory=""|Memory limit|*diff*|
+|--mac-address=""|Container MAC address (e.g. 92:d0:c6:0a:29:33)|YES|
+|--name=""|Assign a name to the container|YES|
+|--net="bridge"|Set the Network mode for the container|YES|
+|-P, --publish-all=false|Publish all exposed ports to random ports|*maybe*|
+|-p, --publish=[]|Publish a container's port(s) to the host|YES|
+|--privileged=false|Give extended privileges to this container|*diff*|
+|--read-only=false|Mount the container's root filesystem as read only|*diff*|
+|--restart="no"|Restart policy (no, on-failure[:max-retry], always)|*maybe*|
+|--security-opt=[]|Security options|*maybe*|
+|-t, --tty=false|Allocate a pseudo-TTY|*diff*|
+|-u, --user=""|Username or UID|*diff*|
+|-v, --volume=[]|Bind mount a volume|*diff*|
+|--volumes-from=[]|Mount volumes from the specified container(s)|*maybe*|
+|-w, --workdir=""|Working directory inside the container|YES|
+
+
+## events
+
+Events **may** may also report ESX and vCenter events.
+
+
+## info
+
+The information provided by *info* appears similar to those reported by Docker but many of them are specific to vSphere.
+
+## inspect
+
+Some parameters **returned** by *inspect* are specific to vSphere.  Others are not relevant.
+
+### Response:
+
+The response for inspect is a JSON document composed of several inner documents.  The following are the primary subdocuments that differs.  For the rest of the response, please visit,
+```
+https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#inspect-a-container.
+```
+
+**AppArmorProfile**
+
+VIC containers isolation unit is a VM.  Processes running in a VIC container do not use AppArmor profiles.
```

Comment:
A VIC container isolation unit is a VM.  Processes running in a VIC container do not use AppArmor profiles.